OpenRA.Mods.Common/Widgets/Logic/Editor/TileSelectorLogic.cs

```diff
@@ -50,11 +52,31 @@ public TileSelectorLogic(Widget widget, WorldRenderer worldRenderer)
 
 			tileCategorySelector.Text = categories.First();
 			IntializeTilePreview(widget, worldRenderer, tileset, categories.First());
+
+			var searchTextField = widget.GetOrNull<TextFieldWidget>("SEARCH_TEXTFIELD");
```

Comment:
Bonus points if you then showed the type of the current mouseover tile just below the `COORDINATE_LABEL`.  Mappers could then look up an existing tile on the map by ID without having to hunt through the categories for it.